dist/oncogrid-debug.js

```diff
@@ -3514,8 +3514,8 @@ OncoHistogram.prototype.renderAxis = function (topCount) {
 
```

Comment:
Thoughts on removing the build files from git? they bloat the diffs quite a bit and for anyone needing the build from a cdn we could point them to something like unpkg.com.